src/collections/resources/tutorials/introduction-to-all-layer5-repositories/index.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: "An introduction to all Layer5 repositories"
+thumbnail: ./all-repos-layer5.png
+category: Service Mesh
+tags: 
+  - Meshery
+  - Tutorial
+author: Lee Calcote
+type: Tutorial
+featured: false
+published: true
+resource: true
+---
+
+import { ResourcesWrapper } from "../../Resources.style.js";
+
+<ResourcesWrapper> 
+        
+        <iframe src="https://docs.google.com/document/d/e/2PACX-1vQEnQ6m6uvAYPFLlBdKQLoB36YKrHh3A7EBSUVfMkSleoDagO4sa2WYg0tDQk51Su6gloAwHUhijO6W/pub?embedded=true"	
```

Comment:
This doesn't look good inside an `iframe`. Instead, just add a brief note to check out the `Repository Overview` and link to https://layer5.io/community/handbook/repository-overview

Reply:
@YashKamboj After the recording, you can just add a button like we have for any [event](https://layer5.io/community/events/project-office-hours-service-mesh-performance-at-kubecon-na-2021) to say "Check out the Layer5 Repository Overview"